src/state/store.ts

```diff
@@ -110,7 +115,7 @@ export const createStore = () =>
     reducer: persistedReducer,
     enhancers: existingEnhancers => {
       // Add the autobatch enhancer to the store setup
-      return existingEnhancers.concat(autoBatchEnhancer())
+      return existingEnhancers.concat(autoBatchEnhancer({ type: 'tick' }))
```

Comment:
use default here without a good reason to use microtasks?

Reply:
https://redux-toolkit.js.org/api/autoBatchEnhancer

Reply:
f0042dd34